FHEM/00_SIGNALduino.pm

```diff
@@ -4272,31 +4385,38 @@ USB-connected devices (SIGNALduino):<br>
   <li>LaCrossePairForSec</li>
   (Only with CC1101 receiver)<br>
   Enable autocreate of new LaCrosse sensors for x seconds. If ignore_battery is not given only sensors sending the 'new battery' flag will be created.<br><br>
-  <li>freq / bWidth / patable / rAmpl / sens / registers<br>
+  <li>cc1101_bWidth / cc1101_dataRate / cc1101_deviatn / cc1101_freq / cc1101_patable / cc1101_rAmpl / cc1101_reg / cc1101_sens<br>
     (Only with CC1101 receiver)<br>
     Set the sduino frequency / bandwidth / PA table / receiver-amplitude / sensitivity<br>
 
     Use it with care, it may destroy your hardware and it even may be
     illegal to do so. Note: The parameters used for RFR transmission are
     not affected.<br>
     <ul>
-      <a name="cc1101_freq"></a>
-      <li><code>cc1101_freq</code> sets both the reception and transmission frequency. Note: Although the CC1101 can be set to frequencies between 315 and 915 MHz, the antenna interface and the antenna is tuned for exactly one frequency. Default is 433.920 MHz (or 868.350 MHz). If not set, frequency from <code>cc1101_frequency</code> will be used.
-      </li>
       <a name="cc1101_bWidth"></a>
       <li><code>cc1101_bWidth</code> can be set to values between 58 kHz and 812 kHz. Large values are susceptible to interference, but make possible to receive inaccurately calibrated transmitters. It affects tranmission too. Default is 325 kHz.
       </li>
+      <a name="cc1101_dataRate"></a>
+      <li><code>cc1101_dataRate</code> , can be set to values ​​between 0.0247955 kBaud and 1621.83 kBaud.
+      </li>
+      <a name="cc1101_deviatn"></a>
+      <li><code>cc1101_deviatn</code> , can be set to values ​​between 1.586914 kHz and 380.859375 kHz.
```

Comment:
Können wir noch hinschreiben was das eigentlich ist oder soll /muss das in der CC1101 Doku nachgelesen werden?

Reply:
>Können wir noch hinschreiben was das eigentlich ist

@sidey79, mein Vorschlag: 
`difference between the minimum and maximum extent of a frequency modulated signal`
@elektron-bbs siehst du das genau so?


>soll /muss das in der CC1101 Doku nachgelesen werden?

@sidey79, es handelt sich um eine notwendige Registereinstellung vom cc1101 um das Signal zu empfangen ggf zu senden
Aus diesem Grund sehe ich es als notwendig an.

Reply:
Mir ging es nicht um das vorhandensein des Attributes sondern um die Funktion von deviatn